lib/widgets/checkout-dialog.coffee

```diff
@@ -0,0 +1,66 @@
+# coffeelint: disable=max_line_length
```

Comment:
I wasn't sure whether the presence of dialog.js indicated a desire to move away from coffee script.  If so I've made it more difficult.  I don't have any experience w/ coffee script so if you intend to move away from it then I'd be happy to refactor this.  Might not get to it for a little while though.